harness/determined/deploy/gke/cli.py

```diff
@@ -0,0 +1,551 @@
+                + "  scheduler, or specifying a different value for --max-gpu-nodes."
+            ).format(args.max_gpu_nodes),
+            "yellow",
+        )
+    if args.cpu_coscheduler:
+        cprint(
+            (
+                "Autoscaling is not supported with the lightweight coscheduling plugin and so a CPU"
+                + " node pool with {} nodes will be statically allocated. This can be changed by"
+                + "  using the default Kubernetes scheduler, or specifying a different value for"
+                + " --max-cpu-nodes."
+            ).format(args.max_cpu_nodes),
+            "yellow",
+        )
+    create_cluster(args)
+    configure_helm(args)
+    if args.cpu_coscheduler:
+        cprint(
+            (
+                "The Priority-Based Gang Scheduler has been enabled for the CPU Agent Pool. To use"
+                + " this feature, please specify the following in your experiment config for"
+                + " commands or CPU Experiments:"
+            ),
+            "yellow",
+        )
+        exp_config = {
+            "environment": {
+                "podspec": {
+                    "metadata": {
+                        "labels": {
+                            "pod-group.scheduling.sigs.k8s.io/name": "<unique task name>",
+                            "pod-group.scheduling.sigs.k8s.io/min-available": "<# of Nodes required>",
+                        }
+                    }
+                }
+            }
+        }
+        cprint(yaml.dump(exp_config), "blue")
+        cprint("Replacing the text in the angle brackets (<>).", "yellow")
+    cmd = ["helm", "install", "determined-gke", args.helm_dir]
+    subprocess.check_call(cmd)
+
+
+def handle_down(args: argparse.Namespace) -> None:
+    validate_location(args.region, isZone=False)
+    cprint(
+        (
+            "Setting kubectl config to cluster {}. Please make sure to run\n`kubectl config "
+            + "set-cluster <other_cluster_name>`\nto interact with other deployed clusters."
+        ).format(args.cluster_name),
+        "yellow",
+    )
+    cmd = ["kubectl", "config", "set-cluster", args.cluster_name]
+    subprocess.check_call(cmd)
+    cmd = ["helm", "uninstall", "determined-gke"]
+    subprocess.check_call(cmd)
+    cmd = [
+        "gcloud",
+        "container",
+        "clusters",
+        "delete",
+        args.cluster_name,
+        "--region",
+        args.region,
+        "--quiet",
+    ]
+    subprocess.check_call(cmd)
+    print("Succesfully deleted GKE Cluster {}".format(args.cluster_name))
+
+
+args_description = Cmd(
+    "gke",
+    None,
+    "GKE help",
+    [
+        Cmd(
+            "up",
+            handle_up,
+            "create gke cluster",
+            [
+                ArgGroup(
+                    "required named arguments",
+                    None,
+                    [
+                        Arg(
+                            "--cluster-name",
```

Comment:
IIRC we've been using `--cluster-id` for `det deploy gcp|aws`, can we do the same here?